dart/lib/src/event_processor/deduplication_event_processor.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:async';
+import 'dart:collection';
+import '../event_processor.dart';
+import '../protocol.dart';
+import '../sentry_options.dart';
+
+const eventsToKeepForDeduplication = 20;
```

Comment:
I'd suggest something like 5

The use case is when there's a:

```
Sentry.captureException(e);
throw e;
```
And the caller also does `captureException` so these usually happen one after the other.

Reply:
I'll make it configureable and set the default value to 5.